packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -167,6 +169,38 @@ class FileAccessLog extends AccessLog {
   }
 }
 
+/**
+ * Represents the properties needed to define client policy
+ */
+export interface TlsClientPolicy {
+  /**
+   * Represents the object for client's TLS certificate
+   *
+   * @default: none
+   */
+  readonly tlsCertificate?: TlsCertificate;
```

Comment:
This is technically part of mTLS

Reply:
Acknowledged.  Will remove this part from the code change to isolate the changes for actual mTLS implementation.